helpers/ExperimentDetails_helper.js

```diff
@@ -0,0 +1,64 @@
+const Helper = require('@codeceptjs/helper');
+
+class ExperimentDetails extends Helper {
+
+    // before/after hooks
+    /**
+     * @protected
+     */
+    _before() {
+        // remove if not used
+    }
+
+    /**
+     * @protected
+     */
+    _after() {
+        // remove if not used
+    }
+
+    async getCurrentActiveKeys() {
```

Comment:
Is there a file missing? I'm not seeing you use these helpers in the existing tests

Reply:
As per our chat, this is the first use of the helpers. I'll use them to refactor the other tests.